app/components/topbar/banners/TestnetWarningBanner.scss

```diff
@@ -1,19 +1,5 @@
-.testnetWarningClassic {
```

Comment:
You didn't port over the differences for YoroiClassic on this one. Is this on purpose?

Reply:
Yes, on purpose.
I made simpler, there is on difference in both `YoroiClassic` and `YoroiModern`